blog/diary/_posts/2025-11-24-cant-ai-rizz-on-command.md

```
@@ -0,0 +1,78 @@
+
+## Commands Just Ain't Rules
+
+### Directory Structure is Part of the Command
+
+A Cursor Command located in `./cursor/commands/foo.md` can be triggered by typing `/foo`.
+
+A Cursor Command located in `./cursor/commands/foo/bar.md` can be triggered by typing `/foo/bar`
+
+This meant that the pattern `ai-rizz` used to ensure its managed rules stayed separate from other rules, of "claiming" two subdirectories of the `.cursor/rules` directory for itself:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)